<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>761</width>
      <height>571</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Model Builder</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>23</y>
        <width>383</width>
        <height>60</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="choose_ticker_label">
           <property name="text">
            <string>Default daily stock data:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ticker_combobox"/>
         </item>
         <item>
          <widget class="QLabel" name="or_label">
           <property name="text">
            <string> or</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="importcsv_label">
           <property name="text">
            <string>Import CSV:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fileexplorer_button">
           <property name="text">
            <string>File Explorer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="currentdata_label">
           <property name="text">
            <string>Current Data: None</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="import_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Import</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="back_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Back</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="graph_placeholder_widget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>511</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>490</y>
        <width>303</width>
        <height>26</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="timewindow_label">
         <property name="text">
          <string>Trading Days:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="window10_button">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="window50_button">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>50</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="window100_button">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="window500_button">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>500</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="window1000_button">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>1000</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="windowmax_button">
         <property name="maximumSize">
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Max</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>30</y>
        <width>82</width>
        <height>78</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="createmodel_button">
         <property name="text">
          <string>Create Model</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="forecast_button">
         <property name="text">
          <string>Forecast</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="forecast_progress_label">
         <property name="text">
          <string>Forecasting...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>145</red>
         <green>145</green>
         <blue>145</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>111</red>
         <green>111</green>
         <blue>111</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>145</red>
         <green>145</green>
         <blue>145</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>111</red>
         <green>111</green>
         <blue>111</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>111</red>
         <green>111</green>
         <blue>111</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>111</red>
         <green>111</green>
         <blue>111</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
